src/index.js

```diff
@@ -1 +1,40 @@
 import './style.css';
+import addScore from './modules/Addscores.js';
+
+const refreshBtn = document.getElementById('refresh');
+const submitBtn = document.getElementById('submit');
+const insertedName = document.getElementById('inserted-name');
+const insertedScore = document.getElementById('inserted-score');
+const fetchedList = document.getElementById('fetched-list');
+
+// let scoreBoard = [];
```

Comment:
- [ ] Kindly remove commented code. It is not a good practice as it can make code hard to read.